api.py

```diff
@@ -45,3 +45,18 @@ def get_most_recent_event():
         {"content_type": "pastEvents", "limit": 1, "order": "-fields.index"}
     )
     return past_events[0]
+
+
+def get_random_resource():
```

Comment:
Since this is now getting all resources instead of a random resource, should rename this function to `get_media_resources`